packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -218,6 +219,8 @@ class CupertinoAlertDialog extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final CupertinoLocalizations localizations = CupertinoLocalizations.of(context)
```

Comment:
I would remove the `??`, we should be providing the default localizations anyway.

Reply:
It looks like `localizations` is always null. Is there any more work that needs to be done to initialize it correctly? Looks like this eventually rolls up to [`inheritFromWidgetOfExactType`](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/framework.dart#L1925), which should be callable in `build()` from what I can tell.

Reply:
Ahh you are right - it looks like we do not provide it anywhere yet